owncloudData/src/main/java/com/owncloud/android/data/spaces/datasources/implementation/OCLocalSpacesDataSource.kt

```diff
@@ -104,7 +108,7 @@ class OCLocalSpacesDataSource(
                         used = spaceQuotaEntity.used,
                     )
                 },
-                root = space.root!!.let { spaceRootEntity ->
+                root = space.root.let { spaceRootEntity ->
```

Comment:
If root is not nullable, `let` is not needed here, don't you think?

Reply:
Not necessary but I think it improves readability, if `space.root` had a longer name it wouldn't be nice to repeat it in every line I think